.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish to NPM
+
+on:
+  release:
+    tags: 
+      - '**-[0-9].[0-9].[0-9]'
+      - '**-[0-9].[0-9].[0-9]-*'
```

Comment:
@krishanthisera I just noticed this. Not sure if actions follow standard regex? But this if it does, this will only capture single digits e.g. `1.2.1` but not `1.12.10`

Reply:
Ohh yeah good spot. I think we may have to fix this :)